apps/greenhouse/package.json

```diff
@@ -15,6 +15,12 @@
   "private": true,
   "devDependencies": {
     "@cloudoperators/juno-config": "*",
+    "@cloudoperators/juno-app-doop": "workspace:*",
```

Comment:
This is why it does not work :) it is a pnpm if you want to revert it please delete this syntax